spec/menu_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "menu"
+describe Menu do
+
+  describe "#view_menu" do
```

Comment:
Did you also test that the view_menu method is actually returning what you expect it should return, ie the list of dishes with prices?
Same for the add_to_order method. Did you test that is returns an array with the items that are selected?